cob_twist_controller/include/cob_twist_controller/hardware_interface_types/hardware_interface_type.h

```diff
@@ -80,19 +81,13 @@ class HardwareInterfacePosition : public HardwareInterfaceBase
     public:
         HardwareInterfacePosition(ros::NodeHandle& nh,
                                   const TwistControllerParams& params)
-        : HardwareInterfaceBase(nh, params),
-          iteration_counter_(0)
+        : HardwareInterfaceBase(nh, params)
         {
-            for(int i = 0; i < params.dof; i++)
-            {
-                ma_.push_back(MovingAverage());
-                vel_support_integration_point_.push_back(0.0);
-                vel_first_integration_point_.push_back(0.0);
-            }
+            ma_.assign(params.dof, MovingAverage(3));
```

Comment:
@ipa-fxm-cm...chose this to be the default in MovingAverage...I just added the number here to make clear that this can be parameterized....